src/components/dashboard/tests/DashboardPage.test.jsx

```diff
@@ -473,12 +520,12 @@ describe('<Dashboard />', () => {
       expect(screen.queryByText(SUBSCRIPTION_EXPIRED_MODAL_TITLE)).toBeTruthy();
     });
 
-    it('should not render when 0 >= daysUntilExpiration and expiration messages are disabled ', () => {
+    it('should not render when 0 >= daysUntilExpirationIncludingRenewals and expiration messages are disabled ', () => {
       useSubscriptions.mockReturnValue({
         data: {
           showExpirationNotifications: false,
           subscriptionPlan: {
-            daysUntilExpiration: 0,
+            daysUntilExpirationIncludingRenewals: 0,
```

Comment:
nit: perhaps include `isCurrent: false`?

```suggestion
            daysUntilExpirationIncludingRenewals: 0,
            isCurrent: false,
```